server/src/main/java/org/elasticsearch/action/admin/indices/stats/ShardStats.java

```diff
@@ -26,37 +26,58 @@
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.io.stream.Streamable;
 import org.elasticsearch.common.io.stream.Writeable;
-import org.elasticsearch.common.xcontent.ToXContent.Params;
 import org.elasticsearch.common.xcontent.ToXContentFragment;
 import org.elasticsearch.common.xcontent.XContentBuilder;
 import org.elasticsearch.index.engine.CommitStats;
+import org.elasticsearch.index.seqno.RetentionLeaseStats;
 import org.elasticsearch.index.seqno.SeqNoStats;
 import org.elasticsearch.index.shard.ShardPath;
 
 import java.io.IOException;
 
 public class ShardStats implements Streamable, Writeable, ToXContentFragment {
+
     private ShardRouting shardRouting;
     private CommonStats commonStats;
     @Nullable
     private CommitStats commitStats;
     @Nullable
     private SeqNoStats seqNoStats;
+
+    @Nullable
+    private RetentionLeaseStats retentionLeaseStats;
+
+    /**
+     * Gets the current retention lease stats.
+     *
+     * @return the current retention lease stats
+     */
+    public RetentionLeaseStats getRetentionLeaseStats() {
```

Comment:
move this further down to the other getters?

Reply:
It is a matter of style? Personally I prefer it that the getters/setters are right next to the corresponding field. I find it easier to work with and maintain. I am happy to put this entire file in that style as a predecessor change if you would prefer.

Reply:
sure, but I think we should stick to the style that the files have. I find the existing style here (and across 99% of the code base) far superior than colocating getters and fields, as it introduces too much noise at the top of the class where I'm typically interested in finding out what fields exist. Your choice.

Reply:
I use ⌘7(Structure) in IntelliJ for that because I also want to see fields from base classes, and you can't see those in the source file. And the top of our files are often littered with static fields, methods, and settings anyway.